lib/statsig_logger.rb

```diff
@@ -15,10 +15,11 @@ def initialize(network, options)
       @options = options
 
       @logging_pool = Concurrent::ThreadPoolExecutor.new(
-        min_threads: [2, Concurrent.processor_count].min,
-        max_threads: [5, Concurrent.processor_count].min,
+        name: 'statsig-logger',
+        min_threads: @options.logger_threadpool_size,
+        max_threads: @options.logger_threadpool_size,
```

Comment:
both min and max are reading the same config - should we allow them to be configured independently?